monsteer/qt/files.cmake

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (c) 2015, Jafet Villafranca Diaz <jafet.villafrancadiaz@epfl.ch>
+#
+# This file is part of Monsteer <https://github.com/BlueBrain/Monsteer>
+#
+
+if(NOT ZEQ_FOUND OR NOT QT5WIDGETS_FOUND)
+  return()
+endif()
+
+set(MONSTEERQT_PUBLIC_HEADERS
+  qt/steeringWidget/SteeringWidget.h)
```

Comment:
I don't like camel case in folder names, this makes the includes a bit awkward IMHO. I am not sure if we really need the steeringWidget subfoler, but if keep it I would vote for all lowercase.. opinions?